lib/extended_git/where_is.rb

```diff
@@ -0,0 +1,66 @@
+module ExtendedGit
+  class WhereIs
```

Comment:
It's hard from me to tell just from the included specs how this class is intended for use. Some top level documentation might be helpful.

Reply:
I added some more documentation, but I think I should add more. I'll try to do it in a later PR.

Reply:
very nice!